### PR TITLE
Pb 728 pybytes releases documentation

### PR DESCRIPTION
**Pull request by elmasria**
* add how to create new release 
* how to deploy new release

### CODE REVIEW

File: content/pybytes/releases/deploy.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: "Deploy Release"
+aliases:
+---
+
+In this section, we will explain to you how to deploy a release.
```
Comment (lilycey) [inline content/pybytes/releases/deploy.md]: 'We will explain how to deploy a release'

**Comment by jirikrepl** [comment]
Is it possible to link example zip archive into the docs?

**Comment by elmasria** [comment]
I am working on adding a link for example `zip` file

**Comment by jirikrepl** [comment]
Let's use code e.g. `const x = 2` only for code snipets.

Schermafbeelding 2019-10-03 om 11 24 30

**Comment by jirikrepl** [comment]
Schermafbeelding 2019-10-03 om 11 32 38


I would like if you can rewrite this file tree using markdown like this:


    my_release.zip
    ├── flash <-- updates files on the /flash partition
    │   └── main.py
    │   └── my_awesome_module.py 
    │   └── ...
    └── sd <-- updates files on the MicroSD card
        └── my_another_awesome_module.py
        └── ...

I got it from this answer
https://stackoverflow.com/a/47510993/1303387

**Comment by elmasria** [comment]
> Let's use code e.g. `const x = 2` only for code snipets.

What do you mean by this?

**Comment by jirikrepl** [comment]
> > Let's use code e.g. `const x = 2` only for code snipets.
> 
> @jirikrepl What do you mean by this?

I meant only use \`\` for code. Not to make thing bold

File: content/pybytes/releases/_index.md [hunk 2]
```diff
@@ -1,48 +1,19 @@
 ---
-title: "Create New Release"
+title: "Pybytes Project Releases"
 aliases:
 ---
 
-In this section, we will explain how to create a new release.
+### What is the release?
+Each **Pybpytes** project contains list of `devices`. You can update the devices `code`, `network configurations` and/or `firmware` version using a `release` version.
```
Comment (jirikrepl) [inline content/pybytes/releases/_index.md]: I see a typo here Pybpytes

**Pull request merged**